dali/python/nvidia/dali/backend.py

```diff
@@ -51,6 +51,11 @@ def deprecation_warning(what):
                             "The next release will support only 10.2 from 10.x familly. "
                             "Please update your environment to CUDA version 10.2 or newer.")
 
+    if __cuda_version__ < 110:
```

Comment:
Oh, so actually for 102 build it says `__cuda_version__ = 10289`

Reply:
Make sense.